automation/helpers/json_responses.py

```diff
@@ -548,7 +548,7 @@
               "message": {
                 "type": "string",
                 "enum": [
-                  "Error parsing ProtoRPC request (Unable to parse request content: Message CombinedContainer is missing required field oauthcode)"
+                  "The browser (or proxy) sent a request that this server could not understand."
```

Comment:
With the new error messages, it seems like we're losing informative error messaging which is too bad. I understand if that's outside the scope of this ticket though

Reply:
Yeah, unfortunately trying to get Flask and webargs to re-generate the same error message that endpoints and protorpc were generating was too much of a headache so I left it like this. [CA-500](https://broadworkbench.atlassian.net/browse/CA-500) should address this.